docs/src/hooks/useCopyToClipboard.ts

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { useEffect, useState, useCallback } from "react";
+import copy from "copy-to-clipboard";
+
+const useCopyToClipboard = (
+  timer: number | undefined = 3000,
```

Comment:
Why is 2s being increased to 3s? The PR description doesn't say.

Reply:
2s seemed as a too fast for me.

Reply:
Fine by me, just to the description that 3s was a personal choice because it feels better.

Reply:
btw, PR description had `3000ms` :)